perception/compare_map_segmentation/src/distance_based_compare_map_filter/node.cpp

```diff
@@ -12,15 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "compare_map_segmentation/distance_based_compare_map_filter_nodelet.hpp"
+#include "node.hpp"
+
+#include "autoware/universe_utils/ros/debug_publisher.hpp"
+#include "autoware/universe_utils/system/stop_watch.hpp"
```

Comment:
@technolojin  Since `debug_publisher.hpp' and `stop_watch.hpp` are already included in pointcloud_preprocessor.hpp, I think it's unnessary to include them
```suggestion
```

Reply:
@badai-nguyen 
The code will work, but the nested include is not a good practice in my opinion.
https://stackoverflow.com/questions/15630111/c-include-how-does-nested-includes-work

DebugPublisher and StopWatch are used in the code, therefore, the header need to be included 'explicitly'.